core/lib/libtomcrypt/include/tomcrypt_custom.h

```diff
@@ -80,6 +80,10 @@
    #endif
 #define XMEMCMP  memcmp
 #endif
+#ifndef XMEM_NEQ
+#include <string_ext.h>
+#define XMEM_NEQ  buf_compare_ct
+#endif
```

Comment:
How is XMEM_NEQ useful here? You could just `#define mem_neq buf_compare_ct` here. Advantage: no change to the .c files.